runtime/themes/iroaseta.toml

```diff
@@ -0,0 +1,117 @@
+"keyword.storage" = { fg = "red" }
+"keyword.storage.type" = { fg = "red" }
+"keyword.storage.modifier" = { fg = "green" }
+"operator" = { fg = "white" }
+"punctuation" = { fg = "white" }
+"punctuation.delimiter" = { fg = "white" }
+"punctuation.bracket" = { fg = "orange" }
+"punctuation.special" = { fg = "yellow" }
+"string" = { fg = "blue" }
+"string.special" = { fg = "blue" }
+"string.regexp" = { fg = "yellow" }
+"tag" = { fg = "green" }
+"label" = { fg = "blue" }
+"type" = { fg = "white" }
+"type.builtin" = { fg = "blue" }
+"namespace" = { fg = "white" }
+"variable" = { fg = "orange" }
+"variable.builtin" = { fg = "orange" }
+"variable.parameter" = { fg = "yellow" }
+"variable.other.member" = { fg = "green" }
+"diff.plus" = { fg = "green" }
+"diff.delta" = { fg = "brown" }
+"diff.minus" = { fg = "red" }
+
+# Markup
+"markup.heading" = { fg = "normal", modifiers = ["bold"] }
+"markup.list" = { fg = "normal" }
+"markup.bold" = { modifiers = ["bold"] }
+"markup.italic" = { modifiers = ["italic"] }
+"markup.link.url" = { fg = "blue" }
+"markup.link.text" = { fg = "gray" }
+"markup.quote" = { fg = "green", modifiers = ["italic"] }
+"markup.raw" = { fg = "blue" }
+
+# Interface
+"ui.background" = { fg = "normal", bg = "default" }
+"ui.help" = { fg = "normal", bg = "dark" }
+"ui.linenr" = { fg = "gray" }
+"ui.linenr.selected" = { fg = "orange" }
+"ui.menu" = { fg = "gray", bg = "dark" }
+"ui.menu.selected" = { fg = "dark", bg = "blue" }
+"ui.popup" = { fg = "gray", bg = "dark" }
+"ui.selection" = { bg = "smark" }
+"ui.selection.primary" = { bg = "pmark" }
+"ui.statusline" = { fg = "normal", bg = "dark" }
+"ui.statusline.normal" = { fg = "green", bg = "list", modifiers = ["reversed"] }
+"ui.statusline.insert" = { fg = "orange", bg = "list", modifiers = ["reversed"] }
+"ui.statusline.select" = { fg = "blue", bg = "list", modifiers = ["reversed"] }
+"ui.statusline.inactive" = { fg = "gray", bg = "list" }
+"ui.text" = { fg = "normal" }
+"ui.text.focus" = { fg = "green" }
+"ui.window" = { fg = "dark" }
+"ui.cursor" = { modifiers = ["reversed"] }
+"ui.cursor.match" = { fg = "orange", modifiers = ["bold"] }
+"ui.cursor.primary" = { modifiers = ["reversed"] }
+"ui.cursorline.primary" = { bg = "list" }
+"ui.cursorline.secondary" = { bg = "dark" }
+"ui.cursorcolumn.primary" = { bg = "list" }
+"ui.cursorcolumn.secondary" = { bg = "dark" }
+"ui.virtual.ruler" = { bg = "rustic_red" }
+"ui.virtual.whitespace" = { fg = "brown" }
+"ui.virtual.indent-guide" = { fg = "brown" }
+"ui.virtual.inlay-hint" = { fg = "gray"}
+"ui.virtual.wrap" = { fg = "yellow"}
+"ui.gutter" = { bg = "default" }
+"ui.gutter.selected" = { bg = "dark" }
+"ui.virtual.jump-label" = { modifiers = ["reversed"] }
+"ui.bufferline.active" = { fg = "yellow", bg = "rustic_red", modifiers = ["bold"] }
+
+# Diagnostics
+"error" = { fg = "red" }
+"warning" = { fg = "brown" }
+"info" = { fg = "blue" }
+"hint" = { fg = "white" }
+"diagnostic" = { fg = "yellow", bg = "#3d0c02", modifiers = ["default"] }
```

Comment:
There isn't a `default` modifier, I think this can just be set to the empty list